virtual_rainforest/models/animals/animal_cohorts.py

```diff
@@ -105,55 +108,98 @@ def metabolize(self, dt: timedelta64) -> None:
         energy_needed = self.metabolic_rate * float((dt / timedelta64(1, "s")))
         self.stored_energy -= min(self.stored_energy, energy_needed)
 
-    def eat(self, food: PlantCommunity) -> float:
+    def herbivory(self, plant_food: PlantCommunity, soil: PalatableSoil) -> float:
         """The function to transfer energy from a food source to the animal cohort.
 
+        The flow to soil here will need to be replaced with a flow to a plant litter
+        pool.
+
         Args:
-            food: The targeted PlantCommunity instance from which energy is
+            plant_food: The targeted PlantCommunity instance from which energy is
                 transferred.
+            soil: The soil community to which plant waste from consumption (leftovers)
+                is deposited for decomposition.
 
         Returns:
-            A float containing the amount of energy consumed in the foraging bout.
+            A float containing the amount of energy consumed in the herbivory bout.
         """
-        consumed_energy = min(food.energy, self.intake_rate * food.energy_density)
+        consumed_energy = (
+            min(
+                plant_food.energy,
+                self.intake_rate * plant_food.energy_density * self.individuals,
+            )
+            * self.individuals
```

Comment:
Aren't you considering self.individuals twice if the winner option is the second one?

Reply:
Oops!